app/controllers/root_controller.rb

```diff
@@ -0,0 +1,6 @@
+# app/controllers/root_controller.rb
+class RootController < ApplicationController
+  def index
+    @random_greeting = Greeting.order('RANDOM()').first
```

Comment:
- [ ] As your previous code reviewer, mentioned you shouldn't need to add any kind of query inside the `index` endpoint because it will just handle to serve your `root` javascript build file. Kindly make your `index` method an empty method, which returns nothing. Although your application is working fine, still it was a `required` change given by your previous code reviewer, so that is why you need to fix it too.